mkt/webapps/tasks.py

```diff
@@ -744,3 +744,26 @@ def populate_is_offline(ids, **kw):
     for webapp in Webapp.objects.filter(pk__in=ids).iterator():
         if webapp.guess_is_offline():
             webapp.update(is_offline=True)
+
+
+@task
+@write
+def adjust_categories(ids, **kw):
+    ADJUSTMENTS = {
+        'books': 'books-comics',
+        'news-weather': 'news',
+    }
+    for chunk in chunked(ids, 100):
+        for app in Webapp.objects.filter(pk__in=chunk):
+            cats = app.categories
```

Comment:
Since it's a `list`, it won't be copied to `cats`, instead `cats` will be a reference to `app.categories`. I'd just remove the intermediary variable, it's not very useful - you don't display the old categories list anywhere so copying would be useless.